motioneye/templates/main.html

```diff
@@ -161,6 +161,11 @@
                         <span class="minimize open"></span>
                     </div>
                     <table class="settings">
+                        <tr class="settings-item" strip="true">
+                            <td class="settings-item-label"><span class="settings-item-label">{{ _("Lingvo") }}</span></td>
```

Comment:
Let's add the translation for `Lingvo` directly. For German language it's `Sprache` 🙂.